submissions/gorodiskiy/HTML-CSS-Popup/css/main.css

```diff
@@ -0,0 +1,187 @@
+* {
+    box-sizing: border-box;
+    -webkit-appearance: none;
+    outline-color: #1E7B0B;
+}
+
+body {
+    padding: 0;
+    margin: 0;
+    color: #000;
+    background-color: #E5E5E5;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+a {
+    color: inherit;
+    text-decoration: none;
+}
+
+ul {
+    padding: 0;
+    margin: 0;
+    list-style: none;
+}
+
+label {
+    cursor: pointer;
+}
+
+input[type="search"] {
+    width: 100%;
+    box-shadow: 0 1px 6px 0 rgba(32, 33, 36, .28);
+    box-sizing: border-box;
+    cursor: text;
+    font-size: 16px;
+    height: 44px;
+    margin: 0 auto;
+    border-radius: 22px;
+    background: transparent;
+    padding: 2px 22px;
+    border: 1px solid #1E7B0B;
+    background-color: #fff;
+}
+
+label[for="more"] {
+    display: block;
+    text-align: center;
+    padding: 10px 20px;
+    background-color: #1E7B0B;
+    color: #fff;
+}
+
+input[type="checkbox"] {
+    display: none;
+}
+
+/* input[type="search"]:focus {
+    background-color: #fff;
+} */
+
+.container {
+    display: flex;
+    justify-content: flex-end;
+    padding: 0 20px;
+}
+
+header {
+    padding: 5px 0;
+}
+
+#open-box:checked~.box {
```

Comment:
Avoid styling by `id`. 
Separate responsibilities: HTML for markup, CSS for visual representing, ID - for javascript